definitions/npm/firebase_v4.x.x/flow_v0.46.x-/firebase_v4.x.x.js

```diff
@@ -0,0 +1,410 @@
+/* @flow */
+
+/**** firebase ****/
+
+declare interface FirebaseConfig {
+  apiKey: string;
+  authDomain?: string;
+  databaseURL: string;
+  projectId: string;
```

Comment:
Just wondering is projectId needed? Inside https://firebase.google.com/docs/web/setup it doesn't mention projectId